styles/styles.css

```diff
@@ -152,7 +173,7 @@ main .section.center p {
   border: 1px solid var(--body-color);
   text-align: center;
   vertical-align: middle;
-  padding: 7px 25px;
+  padding: 13px 25px;
```

Comment:
Why'd you add padding here?

Reply:
Because I took out the line height for the button label.  It made more sense here.